### PR TITLE
Fix #689. MyDownloads option Navigation Drawer

### LINKED ISSUE #689: Lowfi MyDownloads option Navigation Drawer
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/42d3b9aa-8cb7-4ff0-b375-42117c3aed7e/PM-EN-Play-Tab-Side-Menu-

MyDownloadsActivity has already been created but has not been linked to NavigationDrawer, we need to link that to NavigationDrawer.

Icon Link: https://drive.google.com/open?id=1CDqhmWKERmU_34_OIJh0_II5bezWozkJ

Also, add test cases to check whether on click on MyDownloads opens that activity or not.

### PR DESCRIPTION
**Pull request by SayantanBanerjee16**
## Explanation

Fixes #689 
My Downloads option is placed in the navigation drawer and upon clicking it MyDownloads activity opens. 

1) First added the external icon in drawable and then add item in the navigation_drawer_menu.xml.
2) Then added DOWNLOADS(R.id.nav_my_downloads) in the enum class NavigationDrawerItem.
3) Then in NavigationDrawerFragmentPresenter.kt open the Downloads Activity on clicking the item in the drawer and made its status as selected.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/menu/navigation_drawer_menu.xml [hunk 1]
```diff
@@ -11,6 +11,10 @@
       android:id="@+id/nav_help"
       android:icon="@drawable/ic_help_grey_icon_24dp"
       android:title="@string/menu_help" />
+    <item
+      android:id="@+id/nav_my_downloads"
+      android:icon="@drawable/ic_folder_open_24px"
```
Comment (rt4914) [inline app/src/main/res/menu/navigation_drawer_menu.xml]: the file name should be `ic_folder_open_24dp` instead of px.

**Review by rt4914 — CHANGES_REQUESTED** [review]
The PR looks good, but we need some changes.
1. Once you open `MyDownloadsActivity`, if you click on back, the ProfileChooser activity appears, instead home activity should appear. For, reference you can  check the code of `AdministratorControlsActivity`

2. We need to add test cases to `NavigationDrawerTestActivityTest` to make sure that your functionality is correct always. For reference check test cases in this file mainly related to `AdministratorControlsActivity`

**Comment by SayantanBanerjee16** [comment]
> The PR looks good, but we need some changes.
> 
> 1. Once you open `MyDownloadsActivity`, if you click on back, the ProfileChooser activity appears, instead home activity should appear. For, reference you can  check the code of `AdministratorControlsActivity`
> 2. We need to add test cases to `NavigationDrawerTestActivityTest` to make sure that your functionality is correct always. For reference check test cases in this file mainly related to `AdministratorControlsActivity`

Done with changes. Please review :)

File: app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt [hunk 2]
```diff
@@ -180,6 +180,15 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
+  @Test
+  fun testNavigationDrawerTestActivity_openNavigationDrawer_selectMyDownloadsMenuInNavigationDrawer_showsMyDownloadsFragmentSuccessfully() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt:185]: Try writing this test case something similar to `testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_clickAdministratorControls_checkOpensAdministratorControlsActivity`

We should check the intent when `My Downloads` is clicked.

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL, changes suggsted.

File: app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt (hunk 2 above)
Comment (SayantanBanerjee16) [reply app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt:185]: > Try writing this test case something similar to `testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_clickAdministratorControls_checkOpensAdministratorControlsActivity`
> 
> We should check the intent when `My Downloads` is clicked.

@rt4914 Modified test. Please review it again.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**